Packages/Sites/Sfi.Kateheo/Resources/Private/TypoScript/Root.ts2

```diff
@@ -29,8 +29,10 @@ prototype(Sfi.Kateheo:Page) < prototype(TYPO3.Neos:Page) {
 		}
 
 		@context.sectionMenu = Menu {
```

Comment:
And also put the menu tempate into `Private/Templates/TypoScriptObjects` and not into partials (because it's not a partial).